### PR TITLE
Install fmt lib.

### PR DESCRIPTION
**Pull request by jpakkane**

### CODE REVIEW

**Comment by xclaesse** [comment]
You probably also want to install headers then.

**Comment by jpakkane** [comment]
Updated.

**Comment by eli-schwartz** [comment]
It should also install a pkg-config file (and generate a fmt-uninstalled.pc for the resulting devenv).

File: subprojects/packagefiles/fmt/meson.build [hunk 1]
```diff
@@ -29,3 +35,9 @@ fmt_header_only_dep = declare_dependency(
   include_directories: 'include',
   compile_args: '-DFMT_HEADER_ONLY',
 )
+
+pkg_mod = import('pkgconfig')
+pkg_mod.generate(fmt_lib,
+  name: 'fmt',
+  filebase: 'fmt',
```
Comment (eli-schwartz) [inline subprojects/packagefiles/fmt/meson.build:42]: Shouldn't the name and filebase be inferred in this case?

**Pull request merged**